src/sargparse/sargparse.rs

```diff
@@ -72,7 +70,6 @@ struct Argument {
     dtype: ArgumentType,
 }
 
-#[derive(Debug)]
```

Comment:
> The only thing that is printed with {:?} is ArgumentType, meaning that
the other structs and enumerations do not need to derive Debug.

Isn't removing this impl for `ArgumentParser` and `InnerData` a breaking change? Both are exposed to the public API. In general I think it is good practise to have a `Debug` implementation available for exposed types.